include/CL/sycl/device.hpp

```diff
@@ -27,6 +27,16 @@
 #include "CL/sycl/device_selector.hpp"
 #include "CL/sycl/platform.hpp"
 
+#ifndef WEAK_ATTRIB_PREFIX
```

Comment:
Or we may have yet-another-global-file somewhere else.
But we can go with this file for now.